README.md

```diff
@@ -50,7 +50,7 @@ start with. Here are 3 common options:
     ```python
     import pydot
 
-    graphs = pydot.graph_from_dot_file('example.dot')
+    graphs = pydot.graph_from_dot_file("example.dot")
```

Comment:
Double quote changes are okay, I don't have a strong opinion about quotes either way and if anything, uniformity in this regard would be better enforced by `Black`

Reply:
I remind that uniformity in the quotes reduces commit diffs - nowdays there is no code outside of git.

Reply:
> uniformity in the quotes reduces commit diffs

As long as people do not touch existing quotes when they make code changes, it should not matter, right? Or you mean when someone makes a code change and at the same time decides to "clean up" the quoting style around that area (as a form of opportunistic refactoring) and then combines it all together in just a single commit? In that case, yes, it would lead to larger diffs and `black` prevents that.